vault/resource_azure_secret_backend_role_test.go

```diff
@@ -81,6 +86,64 @@ func TestAzureSecretBackendRole(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName+".test_azure_roles", "azure_roles.0.role_name", "Reader"),
 					resource.TestCheckResourceAttrSet(resourceName+".test_azure_roles", "azure_roles.0.scope"),
 					resource.TestCheckResourceAttrSet(resourceName+".test_azure_roles", "azure_roles.0.role_id"),
+				),
+			},
+		},
+	})
+}
+
+func TestAzureSecretBackendRole_AzureGroups(t *testing.T) {
+	subscriptionID := os.Getenv("ARM_SUBSCRIPTION_ID")
+	if subscriptionID == "" {
+		t.Skip("ARM_SUBSCRIPTION_ID not set")
+	}
+	tenantID := os.Getenv("ARM_TENANT_ID")
+	clientID := os.Getenv("ARM_CLIENT_ID")
+	clientSecret := os.Getenv("ARM_CLIENT_SECRET")
+	resourceGroup := os.Getenv("ARM_RESOURCE_GROUP")
+
+	resourceName := "vault_azure_secret_backend_role"
+	path := acctest.RandomWithPrefix("tf-test-azure")
+	role := acctest.RandomWithPrefix("tf-test-azure-role")
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
```

Comment:
Do these run in CI?

Reply:
No, I don't think so. I expect it to fail, due to tests being flaky due to the bug in Vault, where role_name gets set as role_id. Curious to know if you see the same thing locally?